src/components/pages/JoinTheSquad/JoinTheSquadContainer.js

```diff
@@ -28,6 +30,18 @@ const JoinTheSquadContainer = ({ LoadingComponent, ...props }) => {
     return () => (isSubscribed = false);
   }, [memoAuthService]);
 
+  /**
+   *  Initializes state before rendering RenderJoinTheSquad component,
+   *  this means images, necessary state, etc will all be pre-loaded
+   */
+  useEffect(() => {
+    getChildTeam(authState, props.child.id).then(res => {
+      props.setMemberId(res[props.child.id]);
+      props.setTeamSubmissions(res);
+    });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
This is fine for now. I think going forward, we can add the functions to the dep array as well as I believe it's better practice (even though the functions don't change).

Reply:
@drew-ross I just changed this to include the dependencies that the useEffect was asking for -- it took just like one line of refactoring. Can you check and see if it looks ok?